core/src/forwarding_stage.rs

```diff
@@ -135,10 +143,13 @@ pub(crate) fn spawn_forwarding_stage(
                 root_bank_cache,
                 data_budget,
             );
-            Builder::new()
-                .name("solFwdStage".to_string())
-                .spawn(move || forwarding_stage.run())
-                .unwrap()
+            SpawnForwardingStageResult {
+                join_handle: Builder::new()
+                    .name("solFwdStage".to_string())
+                    .spawn(move || forwarding_stage.run())
+                    .unwrap(),
+                client_updater: Arc::new(non_vote_client) as Arc<dyn NotifyKeyUpdate + Send + Sync>,
```

Comment:
maybe a type definition for this would be a bit easier to maintain?

Reply:
`dyn NotifyKeyUpdate + Send + Sync` is used all over the codebase. So to do it properly, I need to create a separate PR modifying it everywhere.